packages/bierzo-wallet/src/routes/welcome/index.tsx

```diff
@@ -33,9 +34,16 @@ const Welcome = (): JSX.Element => {
   const toast = React.useContext(ToastContext);
   const classes = useStyles();
 
-  const onGetIdentities = (): void => {
+  const onGetIdentities = async (): Promise<void> => {
```

Comment:
rename `onGetIdentities` because is misleading. I suggest using `onSendRequestToBeSigned`. Also, update the text on line 69.